src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -59,73 +59,89 @@ export const RegistrationFormActions = ({
         )}
-        <Button data-testid="open-support-button" onClick={toggleSupportModal} sx={{ gridArea: 'support-button' }}>
-          {t('common.support')}
-        </Button>
-        {tabNumber < RegistrationTab.FilesAndLicenses ? (
-          <Box
-            sx={{
-              gridArea: 'save-next-button',
-              display: 'grid',
-              gridTemplateAreas: '"save-button next-button"',
-              columnGap: '1rem',
-            }}>
-            <LoadingButton
-              variant="outlined"
-              loading={isSaving}
-              data-testid="button-save-registration"
-              onClick={async () => {
-                await saveRegistration(values);
-                // Set all fields with error to touched to ensure error messages are shown
-                setTouched(setNestedObjectValues(errors, true));
+
+        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'start' }}>
+          <Button data-testid="open-support-button" onClick={toggleSupportModal}>
+            {t('common.support')}
+          </Button>
+        </Box>
+        {!isLastTab ? (
+          <>
+            <Box
+              sx={{
+                gridArea: 'save-button',
+                display: 'flex',
+                justifyContent: 'end',
               }}>
-              {t('common.save')}
-            </LoadingButton>
-            <Button
-              variant="contained"
-              data-testid="button-next-tab"
-              endIcon={<ArrowForwardIcon />}
-              onClick={() => setTabNumber(tabNumber + 1)}>
-              {tabNumber === RegistrationTab.Description && t('registration.heading.resource_type')}
-              {tabNumber === RegistrationTab.ResourceType && t('registration.heading.contributors')}
-              {tabNumber === RegistrationTab.Contributors && t('registration.heading.files_and_license')}
-            </Button>
-          </Box>
+              <LoadingButton
+                variant="outlined"
+                loading={isSaving}
+                data-testid="button-save-registration"
+                onClick={async () => {
+                  await saveRegistration(values);
+                  // Set all fields with error to touched to ensure error messages are shown
+                  setTouched(setNestedObjectValues(errors, true));
+                }}>
+                {t('common.save')}
+              </LoadingButton>
+            </Box>
+            <Box sx={{ gridArea: 'next-button', display: 'flex', justifyContent: 'end' }}>
+              <Tooltip title={t('common.next')}>
+                <IconButton onClick={() => setTabNumber(tabNumber + 1)}>
+                  <KeyboardArrowRightIcon
+                    sx={{
+                      color: 'white',
+                      borderRadius: '50%',
+                      bgcolor: 'primary.light',
+                      height: '1.875rem',
+                      width: '1.875rem',
+                    }}
```

Comment:
Litt usikker på hva som faktisk er best her, men jeg bruker vanligvis å sette fontSize på ikonet i stedet for å sette bredde/høyde 🤔 
```suggestion
                  <KeyboardArrowRightIcon
                    fontSize='large'
                    sx={{
                      color: 'white',
                      borderRadius: '50%',
                      bgcolor: 'primary.light',
                    }}
```

Reply:
IconButton har også en size-property, som bestemmer størrelsen på selve knappen rundt ikonet

Reply:
lar det være slik, slik at størrelsene stemmer overens med figma